lte/gateway/docker/services/build/Dockerfile

```diff
@@ -0,0 +1,91 @@
+FROM ubuntu:20.04 AS builder
+
+ENV DEBIAN_FRONTEND=noninteractive
```

Comment:
It appears that setting DEBIAN_FRONTEND for env is not a best practice since it lingers even after the build itself is complete. Using ARG is recommended.

Reply:
I will follow scott's MME dockerfile and move the DEBIAN_FRONTEND=noninteractive in front of the apt-get install command